app/graphql/types/query_type.rb

```diff
@@ -23,6 +23,24 @@ def all_partners
       Partner.all
     end
 
+    ####
+
+    field :event, EventType, "Find Event by ID" do
+      argument :id, ID
+    end
+
+    field :all_events, [EventType]
+
+    def event(id:)
```

Comment:
is this trailing colon correct? just looks odd :)

Reply:
this i believe is a fancy ruby convenience way of doing `def event(opts={})` followed by `Event.find(opts[:id])`